packages/flutter_test/lib/src/binding.dart

```diff
@@ -1387,6 +1387,11 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
 /// doesn't trigger a paint, since then you could not see anything
 /// anyway.)
 class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
+
+  LiveTestWidgetsFlutterBinding() {
```

Comment:
If it is LiveTestWidgetsFlutterBinding, shouldn't the asset be able to load from the real backend? Why do we need to mock the asset channel?